services/transactionpool/service.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/orbs-network/orbs-network-go/instrumentation/log"
 	"github.com/orbs-network/orbs-network-go/instrumentation/metric"
 	"github.com/orbs-network/orbs-network-go/synchronization"
+	"github.com/orbs-network/orbs-network-go/synchronization/supervized"
```

Comment:
Maybe rename `supervized` to `goroutine`? `supervized.OneOff` doesn't sound like anything having to do with a thread / goroutine / coroutine. `goroutine.OneOff` is a little more clear that this is a threading mechanism.

Another option is to go with `supervized.LongLived` and `supervized.ShortLived` - it also sounds more like threads..

Reply:
Just find me a better name than `supervized.OneOff` - it's not clear what it does